components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -446,7 +446,11 @@ class GeckoEngineSession(
      */
     @Suppress("ComplexMethod")
     private fun createNavigationDelegate() = object : GeckoSession.NavigationDelegate {
-        override fun onLocationChange(session: GeckoSession, url: String?) {
+        override fun onLocationChange(
+            session: GeckoSession,
+            url: String?,
+            perms: List<GeckoSession.PermissionDelegate.ContentPermission>
```

Comment:
Hm the GV docs don't really suggest what we're supposed to do with the new param, or did I miss it? If we're ignoring it for now, maybe we should file and link to a follow-up here?

Reply:
For now we are not going to need this new parameter, as we don't have the API to processed with the migration, they split the work in two parts: 
* Part one: [the initial API to query the sites](https://bugzilla.mozilla.org/show_bug.cgi?id=1654832).
* Part two: [The API for adding/removing sites](https://bugzilla.mozilla.org/show_bug.cgi?id=1699480).

For now we are only removing the old API and waiting for part two for starting the migration process.

Reply:
👍 OK, I think we can close this PR though as the `onLocationChange` override was introduced in a non-breaking way i.e. there's a default impl: https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/GeckoSession.java#3662